packages/vue/src/index.ts

```diff
@@ -227,6 +228,8 @@ const Frontegg: PluginObject<PluginOptions> | any = {
         setStoreKey(this, store);
         this.fronteggAuth = Vue.fronteggAuth;
         this.loginWithRedirect = loginWithRedirect.bind(this);
+        // _entitlements was added for hack purposes - we want the user to pass it to have a reactive part in the computed property, then it will get updated
```

Comment:
I woukld remove this comment as it is readable by our users (we dont want to explicitly tell our clients we use hacks)

instead we can write more informative comment explaining the user what exactly he should pass here

Reply:
pls take a look again